docs/usage/getting-started/use-cases.md

```diff
@@ -200,3 +200,8 @@ The typical workflow for a company is:
 - Set that repository as the default `extends` value when onboarding new repositories
 
 This means that repositories get the centralized config by default, and any changes made to the centralized config repository are propagated to other repositories immediately.
+
+## How others use Renovate
```

Comment:
I though of putting this new section after the `## Development dependency updates` section, but that didn't feel right to me. As then we're linking to another page right after the users lands on the use cases page... 😄